jsoup/src/main/kotlin/Attributes.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.html.jsoup
+
+import org.jsoup.nodes.Attributes
+
+//region Read-Only Properties
+val Attributes.size: Int
+    get() = size()
+//endregion
+
+//region Methods
+operator fun Attributes.set(key: String, value: String)
+    = put(key, value)
+
+operator fun Attributes.set(key: String, value: Boolean)
+    = put(key, value)
+
+operator fun Attributes.contains(key: String): Boolean
+    = hasKey(key)
+
+operator fun Attributes.contains(attribute: Pair<String, String>): Boolean
+    = this.hasKey(attribute.first) && this[attribute.first] == attribute.second
+
+fun Attributes.toMap(): Map<String, String>
+    = associate { it.key to it.value }
```

Comment:
`associateBy( { it.key }, { it.value })` would be more efficient as it doesn't cause `Pair` instantiations